temoa/extensions/myopic/myopic_sequencer.py

```
@@ -256,23 +256,23 @@ def start(self):
             # prep next loop
             last_base_year = idx.base_year  # update
 
-            # delete anything in the OutputObjective table, it is nonsensical...
+            # delete anything in the output_objective table, it is nonsensical...
             self.output_con.execute(
-                f'DELETE FROM OutputObjective WHERE scenario == "{self.config.scenario}"'
+                f'DELETE FROM output_objective WHERE scenario == "{self.config.scenario}"'
```

Ruff (0.14.3)

Possible SQL injection vector through string-based query construction

(S608)